decidim-assemblies/app/cells/decidim/assemblies/content_blocks/highlighted_assemblies_cell.rb

```diff
@@ -8,8 +8,6 @@ def highlighted_spaces
           @highlighted_spaces ||= OrganizationPrioritizedAssemblies
                                   .new(current_organization, current_user)
                                   .query
-                                  .with_attached_hero_image
```

Comment:
Without this we had bullet warnings

https://github.com/decidim/decidim/actions/runs/8342208142/job/22829980082

```
1st Try error in ./spec/requests/assembly_search_spec.rb:39:
user: runner
 
AVOID eager loading detected
  Decidim::Scope => [:scope_type]
  Remove from your query: .includes([:scope_type])
Call stack
  /home/runner/work/decidim/decidim/decidim-core/app/helpers/decidim/check_boxes_tree_helper.rb:170:in `scope_children_to_tree'
  /home/runner/work/decidim/decidim/decidim-core/app/helpers/decidim/check_boxes_tree_helper.rb:159:in `block in filter_scopes_values_from'
  /home/runner/work/decidim/decidim/decidim-core/app/helpers/decidim/check_boxes_tree_helper.rb:156:in `each'
  /home/runner/work/decidim/decidim/decidim-core/app/helpers/decidim/check_boxes_tree_helper.rb:156:in `flat_map'
  /home/runner/work/decidim/decidim/decidim-core/app/helpers/decidim/check_boxes_tree_helper.rb:156:in `filter_scopes_values_from'
  /home/runner/work/decidim/decidim/decidim-core/app/helpers/decidim/check_boxes_tree_helper.rb:96:in `filter_global_scopes_values'
  /home/runner/work/decidim/decidim/decidim-assemblies/app/helpers/decidim/assemblies/filter_assemblies_helper.rb:24:in `filter_sections'
  /home/runner/work/decidim/decidim/decidim-assemblies/app/views/decidim/assemblies/assemblies/index.html.erb:17:in `block in __home_runner_work_decidim_decidim_decidim_assemblies_app_views_decidim_assemblies_assemblies_index_html_erb__1035453487187093629_448100'
  /home/runner/work/decidim/decidim/decidim-assemblies/app/views/decidim/assemblies/assemblies/index.html.erb:14:in `__home_runner_work_decidim_decidim_decidim_assemblies_app_views_decidim_assemblies_assemblies_index_html_erb__1035453487187093629_448100'
  /home/runner/work/decidim/decidim/decidim-assemblies/app/controllers/decidim/assemblies/assemblies_controller.rb:22:in `block (2 levels) in index'
  /home/runner/work/decidim/decidim/decidim-assemblies/app/controllers/decidim/assemblies/assemblies_controller.rb:18:in `index'
  /home/runner/work/decidim/decidim/decidim-core/app/controllers/concerns/decidim/use_organization_time_zone.rb:21:in `use_organization_time_zone'
  /home/runner/work/decidim/decidim/decidim-core/app/controllers/concerns/decidim/locale_switcher.rb:24:in `switch_locale'
  /home/runner/work/decidim/decidim/decidim-core/lib/decidim/middleware/strip_x_forwarded_host.rb:12:in `call'
  /home/runner/work/decidim/decidim/decidim-core/lib/decidim/middleware/current_organization.rb:22:in `call'
  /home/runner/work/decidim/decidim/decidim-assemblies/spec/requests/assembly_search_spec.rb:32:in `block (2 levels) in <top (required)>'
```